.github/workflows/ci.yml

```diff
@@ -151,6 +151,7 @@ jobs:
       - name: Run e2es
         env: 
           SPA_ENV: ci,b2c
+          BUILD_NUMBER: ci-build-number-${{ github.run-attempt }}
```

Comment:
You mentioned to add Commit ID string instead of run-attemptto make sure it is a unique string. is it still needed?

Reply:
👍

Reply:
fixed 

sorry never had time to actual work on things 

(no more non-relevant/help meetings for me till Monday) unless it's related to pipeline or team alchemyst :P